incorporating-accessibility-into-your-dev-process/demo/src/Audio/Audio.test.ts

```diff
@@ -0,0 +1,7 @@
+import Audio from "./Audio";
+
+describe('Audio component', () => {
+    it('is defined', () => {
+        expect(Audio).toBeDefined;
```

Comment:
This test doesn't do anything, which is fine since this is an exercise, but can we at least render the component to enforce good habits.

`.toBeDefined` is a function

Reply:
the rest of the tests look similar

Reply:
I’ll open up a PR to fix using `toBeDefined` (it was copied from other tests).

Similar to Fractal Modlets, these files are intended to just be placeholders so we have the testing infrastructure set up. For my bit, I will be adding tests for accessibility stuff, but probably not for everything in the component (because that’s not the point of the bit). If we get further down the line and decide we want coverage for _everything_, we can talk about that (but for now it’s not my priority).

Reply:
this in particular is what im worried about `expect(Audio)`. I think its fine that we don't have the infra setup, but it makes it seem like this is an acceptable way to test components and its not.

Reply:
This is good feedback, thank you.

Here’s the “bare minimum” that we have in Fractal Modlets: https://github.com/bitovi/trainings/blob/c52ac5be8c588c5dff44cbd400d31059142a14dc/fractal-modlets/Exercise1/solution/ProductPage/CartButton/CartButton.test.ts#L3-L5

Should our bare minimum be something more like this (checking to see if the component renders)? https://github.com/bitovi/trainings/blob/c52ac5be8c588c5dff44cbd400d31059142a14dc/unit-testing-react-what-why-how/test-rock-paper-scissors/solution/src/RockPaperScissors.test.tsx#L48-L53

Is there anything else you would consider a requirement for every minimal test file that we include?